example/app/data/DemoDataSource/recipes/apps/signal_app/recipe_links/signals_simple/case.recipe.json

```diff
@@ -98,10 +98,30 @@
       }
     },
     {
-      "name": "RunInt",
+      "name": "RunJob",
       "type": "CORE:UiRecipe",
       "description": "running",
-      "plugin": "signal-job-single"
```

Comment:
maybe remove this from `example/src/plugins/job-ui-single/index.tsx` and from `example/src/plugins/index.tsx`?